### DISCUSSION TITLE
Unable to run NIFI scheduler locally

### DISCUSSION THREAD

**Discussion by GitVivekHub** [Sunbird-cQube/community#799] [Q&A]
I am trying to run c-qube locally, the issue that im currently facing is that when i run the "Run Latest code local" scheduler api it gives me an error with config.json file not found.  I have atteched the SS for the same.
![Screenshot from 2023-07-07 12-29-50](https://github.com/Sunbird-cQube/community/assets/136580365/ff4faa67-48a5-4230-8e03-d724d3672275)

**Comment by DevashishRaghav** [comment]
@padmanabha1234 @htvenkatesh

**Comment by RNarmatha** [comment]
@dhanush-2397 - please look into this issue

**Comment by dhanush-2397** [comment]
@GitVivekHub Do you have a .env file in your processing-ms? If not, please create one inside cqube with the following key values.
DATABASE_URL=""
DB_HOST = ''
DB_PORT = 
DB_USERNAME = 
DB_PASSWORD = 
DB_NAME = 
PROGRAM_TYPE = 
Also please ensure that you have the latest code in your local.
This documentation https://docs.google.com/document/d/1Y65HeaHRW4tODCIsizd3KBPG-dv6NlXllnkFkq5unX0/edit?usp=sharing 
can help uou complete the setup.
